cwms-data-api/src/main/java/cwms/cda/data/dto/Project.java

```diff
@@ -0,0 +1,317 @@
+package cwms.cda.data.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV2;
+import java.time.Instant;
+
+@JsonDeserialize(builder = Project.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+@FormattableWith(contentType = Formats.JSON, formatter = JsonV2.class)
+public class Project extends CwmsDTO {
+
+    private final String name;
+    private final Double federalCost;
+    private final Double nonFederalCost;
+    private final Instant costYear;
```

Comment:
Should costYear stay an Instant?  Is it always June 1 ?

Reply:
It's DATE in the database, so it's likely it could vary. I think INSTANT has given us the most flexbility in rendering and reading back.